data/copyright.html

```diff
@@ -335,6 +335,9 @@
 
   <li><a href="https://github.com/mapbox/maki" title="Maki">Maki</a><br>
     &copy; 2022 Mapbox <a href="#cc0-1" class="license">CC0 1.0 Universal</a></li>
+  
+  <li><a href="https://www.onlinewebfonts.com/svg" title="OnlineWebFonts">OnlineWebFonts</a><br>
```

Comment:
This URL shows 404. Where did you find their license?

Reply:
ah oops, it should be https://www.onlinewebfonts.com/icon/

 https://www.onlinewebfonts.com/icon/74710

Reply:
Sorry, I still don't see the license text in any of the links.

Reply:
it is a bit hidden, but when you press download it shows the attribution info, and the license is specified.
![image](https://github.com/organicmaps/organicmaps/assets/26939824/9b759b60-7be4-4e53-b220-ff88b2eb8529)

Reply:
@biodranik is it good to be merged?